gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest_asyncio.py.j2

```diff
@@ -225,6 +225,9 @@ class Async{{service.name}}RestTransport(_Base{{ service.name }}RestTransport):
         return self._{{method.name}}(self._session, self._host, self._interceptor)  # type: ignore
 
     {% endfor %}
+    {% for name, sig in api.mixin_api_signatures.items() %} 
+    {{ shared_macros.generate_mixin_call_method(service, api, name, sig, is_async=True) | indent(4) }}
```

Comment:
This doesn't need to block, but would be a good follow-up: this macro is defining the each `_Mixin` class  next to the corresponding `@property`. The other `_Method` classes are defined all together, and then their `@property` all together. I think it would be good to define the `_mixin` classes right after the `_Method` classes, and the mixin `@property` right after the method `@property`. WDYT? Could you file an issue so we don't forget?

Reply:
I've filed and linked an issue to follow up on this.